api/python/provisioner/commands/deploy_vm.py

```diff
@@ -157,13 +162,16 @@ def _run_states(self, states_group: str, run_args: run_args_type):
             # TODO use salt orchestration
             if setup_type == SetupType.SINGLE:
                 logger.debug("Executing for single node.")
-                if "sync" not in state:
+                if "^sync" not in state:
```

Comment:
@mazinamdar looks like a regex. Do you think it will do what you want? What about `startswith`?

Reply:
oops that is not working as expected, thanks, updated using `startswith`